spark/src/main/java/org/opensearch/sql/spark/storage/implementor/SparkDefaultImplementor.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.storage.implementor;
+
+import lombok.RequiredArgsConstructor;
+import org.opensearch.sql.planner.DefaultImplementor;
+import org.opensearch.sql.spark.storage.SparkScan;
+
+/**
+ * Default Implementor of Logical plan for spark.
+ */
+@RequiredArgsConstructor
+public class SparkDefaultImplementor
+    extends DefaultImplementor<SparkScan> {
```

Comment:
If no override logic, this subclass is not necessary?

Reply:
Thanks for catching this. removed subclass.